hist/hist/src/TH1.cxx

```diff
@@ -4454,6 +4454,26 @@ Double_t TH1::GetEffectiveEntries() const
    return (s[1] ? s[0]*s[0]/s[1] : TMath::Abs(s[0]) );
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Shortcut to set the four histogram colors with a single call.
```

Comment:
Method changes **three** colors - not four

Reply:
good point